polimec-common/common/src/credentials/mod.rs

```diff
@@ -121,6 +122,45 @@ where
 	) -> Result<jwt_compact::Token<Self::Claims>, ValidationError> {
 		let signing_key =
 			<<Ed25519 as Algorithm>::VerifyingKey>::from_slice(&verifying_key).expect("The Key is always valid");
-		Ed25519.validator::<Self::Claims>(&signing_key).validate(&token)
+		let result = Ed25519.validator::<Self::Claims>(&signing_key).validate(&token);
+		result
```

Comment:
I prefer it since it makes it easier to debug. What do you think @lrazovic ?

Reply:
You mean for placing dbg between let result and result return? I think it would be proper to actually directly return the result (probably clippy will also start complaining), and only do a quick rewrite if you want to debug the result and move it back once debug is not necessary anymore

Reply:
I mean for using the debugger, but sure